src/packages/src/xroad/common/secret-store-local/usr/share/xroad/scripts/backup_openbao_db.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+dump_file="$1"
+
+db_connection_url=$(awk '/storage "postgresql"/,/\}/' /etc/openbao/openbao.hcl | grep 'connection_url' | cut -d'"' -f2)
+pat='^postgres:\/\/([^:]+):([^@]+)@([^:]+):([0-9]+)\/([^?]+)(\?(.*))?$'
+if [[ $db_connection_url =~ $pat ]]; then
+  db_conn_user="${BASH_REMATCH[1]}"
+  db_user=${db_conn_user%%@*}
+  db_password="${BASH_REMATCH[2]}"
+  db_addr="${BASH_REMATCH[3]}"
+  db_port="${BASH_REMATCH[4]}"
+  db_database="${BASH_REMATCH[5]}"
+  db_options="${BASH_REMATCH[7]}" # full query string after '?', if present
+  schema_pat='(^|&)search_path=([^&]*)'
+  if [[ $db_options =~ $schema_pat ]]; then
+    db_schema="${BASH_REMATCH[2]}"
+  else
+    db_schema="public"
+  fi
+  pg_options="-c client-min-messages=warning -c search_path=$db_schema"
+else
+  echo "Unable to determine OpenBao PostgreSQL connection URL in /etc/openbao/openbao.hcl"
+  exit 1
+fi
+
+# Reading custom libpq ENV variables
+if [ -f /etc/xroad/db_libpq.env ]; then
+  source /etc/xroad/db_libpq.env
```

Comment:
db_libpq is serverconf specific. We should probably create a new env file for openbao db.

Reply:
db_libpq.env is shared between of serverconf, messagelog & opmonitor databases during their setup (see [setup_database()](https://github.com/nordic-institute/X-Road/blob/develop/src/packages/src/xroad/common/base/usr/share/xroad/scripts/_setup_db.sh#L52-L59) which is invoked by [setup_messagelog_db.sh](https://github.com/nordic-institute/X-Road/blob/develop/src/packages/src/xroad/common/addon/proxy/setup_messagelog_db.sh#L18) & [setup_opmonitor_db.sh](https://github.com/nordic-institute/X-Road/blob/develop/src/packages/src/xroad/common/op-monitor/usr/share/xroad/scripts/setup_opmonitor_db.sh) both alike.
Although not fully sure but this does look like a intended feature to me.
If that's the case I do think db restore and init processes should act consistently in this regard.